packages/core/src/contrib/Constraints.ts

```diff
@@ -278,8 +489,4 @@ export const constrDict = {
   ...constrDictCurves, // Curve-specific constraints.
 };
 
-// `_constrDictVals` causes TypeScript to enforce that every function in
-// `constrDict` returns an `ad.Num`
-const _constrDictVals: ((...rest: never[]) => ad.Num)[] = Object.values(
-  constrDict
-);
+const _constrDictVals: ConstrFunc[] = Object.values(constrDict);
```

Comment:
Can you add back a comment similar to the one that was here before?

Reply:
yes